docs/release-notes/8.4.0-1.md

```diff
@@ -1,2 +1,79 @@
+
+<!--
+## Bug fixes
+
+[PXB-3337](https://perconadev.atlassian.net/browse/PXB-3337): Fixed a memory out of bounds read when `chunk_name.size() < 21` (Thanks to imrivera,
+Ignacio Martínez Rivera, for fixing this issue.)
+
+-------->
+
+## Deprecated or removed
+
+### Changes made in Percona XtraBackup 8.4.0-1
+
+* The terms MASTER and SLAVE have been replaced with SOURCE and REPLICA to align with MySQL 8.4.0.
+
+  Find the full list of removed (MASTER/SLAVE) terminology in the [MySQL Deprecation and Removal Notes](https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-0.html).
+
+### Changes inherited from the Innovation releases
+
+* Percona XtraBackup 8.4.0-1 supports only the following components and plugins:
+
+    Supported components
+
+    * [Keyring Vault component]
+
+    * [KMIP component]
+
+    * [AWS KMS component]
+
+    * [Keyring file component]
+
+    Supported plugin
+
+    * The [Keyring file plugin] is supported but deprecated and may be removed in future releases.
+
+    | Feature                   | Supported | Deprecated |
+    |---------------------------|-----------|------------|
+    | Keyring Vault component   | Yes       | No         |
+    | KMIP component            | Yes       | No         |
+    | AWS KMS component         | Yes       | No         |
+    | Keyring file component    | Yes       | No         |
+    | Keyring file plugin       | Yes       | Yes        |
+
+* The `--stats` mode of operation for the xtrabackup binary has been removed.
+
+## Useful links
+
+Install [Percona XtraBackup 8.4.0-1](..//installation.md)
+
+The [Percona XtraBackup GitHub repository](https://github.com/percona/percona-xtrabackup)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
+
+[Percona XtraBackup 8.0 series]: https://docs.percona.com/percona-xtrabackup/8.0/
+[Percona Server for MySQL 8.0.x]: https://docs.percona.com/percona-server/8.0/
+[8.3.0-1]: https://docs.percona.com/percona-xtrabackup/innovation-release/
+[Quickstart Guide for Percona XtraBackup]: ..//quickstart-overview.md
+[KMIP component]: https://docs.percona.com/percona-server/innovation-release/using-kmip.html
+[AWS KMS component]: https://docs.percona.com/percona-server/innovation-release/using-amz-kms.html
+[Keyring Vault component]: https://docs.percona.com/percona-server/innovation-release/use-keyring-vault-component.html
+[Keyring file component]: https://docs.percona.com/percona-server/innovation-release/use-keyring-file.html?h=#use-the-keyring-file-component
+[Keyring file plugin]: https://docs.percona.com/percona-server/innovation-release/use-keyring-file.html?h=#use-the-keyring-file-plugin
```

Comment:
have to remove the file plugin from 8.4.0-1

Reply:
these is just the list of links used in the doc